universal-react/server/wrapInDocument.js

```diff
@@ -25,7 +25,17 @@ function getManifest() {
   return _manifest;
 }
 
-export default function wrapInDocument(content, appData, helmetContext, cspNonce, inlineStyles = []) {
+export default function wrapInDocument(
+  content,
+  appData,
+  helmetContext,
+  cspNonce,
+  inlineStyles = [],
+  overriddenDocument
```

Comment:
I think you can do here `document = defaultDocument` as it's done for inline styles.
You can remove the L37 then.